packages/api/docs/future/paths/application.yaml

```diff
@@ -0,0 +1,166 @@
+applications:
+  post:
+    summary: >
+      Register a new application to use the Permanent API.
+      Authorization token must belong to a registered developer account.
+    operationId: createApplication
+    security:
+      - bearerHttpAuthentication: []
+    tags:
+      - application
+    requestBody:
+      content:
+        application/json:
+          schema:
+            type: object
+            required: [claims]
+            properties:
+              redirectUris:
+                type: array
+                items:
+                  type: string
+                  format: uri
+              scopes:
+                type: array
+                items:
+                  type: string
+                  enum:
+                    - record.create
```

Comment:
This set of enum values exists in two places (and I think the list is coupled across both, so if one changes the other should too).  I don't think the enum value list can be reused, but the entire enum / type can become a schema definition.

Let's define the items in terms of a referenced schema type of `ApplicationScope` (or whatever name you think is best)